go.mod

```diff
@@ -1,9 +1,8 @@
 module github.com/go-chi/httplog
 
-go 1.14
+go 1.19
 
 require (
 	github.com/go-chi/chi/v5 v5.0.7
-	github.com/rs/zerolog v1.27.0
-	golang.org/x/sys v0.0.0-20220715151400-c0bba94af5f8 // indirect
+	golang.org/x/exp v0.0.0-20230108222341-4b8118a2686a
```

Comment:
Also, while using this fork. I found this to be outdated and the interface changed a bit.

Reply:
oh ill check it out, i think it's best to switch to slog by july, it'll ship out with go 1.21 in august, by then the api should be stable